apps/condo/domains/billing/schema/SumBillingReceiptsService.js

```diff
@@ -50,14 +50,14 @@ const SumBillingReceiptsService = new GQLCustomSchema('SumBillingReceiptsService
                 description: 'Calculate sum of organizations billing receipts by organizationId or tin and period',
                 errors: ERRORS,
             },
-            resolver: async (parent, args) => {
+            resolver: async (parent, args, context) => {
                 const { data: { period, tin, organization, importRemoteSystem } } = args
 
                 if (!tin && !organization) {
-                    throw new GQLError(ERRORS.TIN_OR_ORGANIZATION_ID_MUST_BE_SPECIFIED)
+                    throw new GQLError(ERRORS.TIN_OR_ORGANIZATION_ID_MUST_BE_SPECIFIED, context)
```

Comment:
What about to make the `context` parameter required for `GQLError` constructor?

Reply:
It's required for detecting client language for error translations

Reply:
And for extracting `req.id`.